src/integ_test_resources/ios/sdk/integration/cdk/cdk_integration_tests_ios/textract_stack.py

```diff
@@ -0,0 +1,19 @@
+from aws_cdk import aws_iam, core
+
+from common.common_stack import CommonStack
+from common.platforms import Platform
+from common.region_aware_stack import RegionAwareStack
+
+
+class TextractStack(RegionAwareStack):
+    def __init__(self, scope: core.Construct, id: str, common_stack: CommonStack, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        self._supported_in_region = self.is_service_supported_in_region()
+
+        all_resources_policy = aws_iam.PolicyStatement(
+            effect=aws_iam.Effect.ALLOW, actions=["textract:AnalyzeDocument"], resources=["*"]
+        )
+        common_stack.add_to_common_role_policies(self, policy_to_add=all_resources_policy)
+
+        self.save_parameters_in_parameter_store(Platform.IOS)
```

Comment:
Are there any parameters to store for this stack? I don't see any resources being created.